examples/multiglobe.html

```diff
@@ -110,7 +110,7 @@
                     if (this.t < 1) {
                         globeView.notifyChange(true);
                     } else {
-                        globeView.removeFrameRequester(animator);
+                        globeView.removeFrameRequester(itowns.MAIN_LOOP_EVENTS.BEFORE_RENDER, animator.update);
```

Comment:
If I'm not mistaken, animator uses `this`. You need to bind update to animator for this to work,  or maybe better: make animator a simple function (with t in the outside scope). As you wish :-)

Reply:
Changed it for a global `t` and simpler `animator`.